lib/iris/tests/unit/fileformats/rules/test__make_cube.py

```diff
@@ -18,18 +18,20 @@
 
 from __future__ import (absolute_import, division, print_function)
 from six.moves import (filter, input, map, range, zip)  # noqa
+import six
 
 # Import iris.tests first so that some things can be initialised before
 # importing anything else.
 import iris.tests as tests
 
-from iris.fileformats.rules import _make_cube
-import iris.fileformats.rules
+import warnings
+
+from iris.fileformats.rules import ConversionMetadata, _make_cube
```

Comment:
It just seems a bit daft to duplicate an import line like that, especially when there doesn't seem that much value gained from the style...